### ISSUE TITLE
When using --resource, --extends and --sections are not used.

### ISSUE THREAD

**Issue by mikemand** [svenluijten/artisan-view#2]
I ran:

```
$ php artisan make:view --extends="layouts.app" --sections="title,content" admin.jobs
```

It created the correct files, but they are all blank; no `@extends('layouts.app')` or sections applied.

**Comment by svenluijten** [comment]
Good catch! I'm aware of the issue and working on a solution :smile:

**Comment by svenluijten** [comment]
This is fixed since 1.2.1, grab it while it's hot!
